File: Nodejs/src/utils/response.js
/**
 * Response utility functions
 */

/**
 * Build standardized API response
 * @param {string} message - Response message
 * @param {*} data - Response data
 * @param {number} code - Response code
 * @returns {Object} Standardized response object
 */
const buildResponse = (message, data = null, code = 200) => ({
  code,
  message,
  data
});

module.exports = {
  buildResponse
};

File: Python/utils/response.py
from typing import Any, Dict


def build_response(message: str, data: Any = None, code: int = 200) -> Dict[str, Any]:
    """Build standardized API response object.

    The format follows the README specification:
    {
        "code": 200,
        "message": "操作成功",
        "data": {...}
    }
    """
    return {"code": code, "message": message, "data": data}

